src/componentprocessor/index.ts

```diff
@@ -21,6 +21,14 @@ interface ComponentField {
     [key: string]: FieldAttributes;
 }
 
+interface FunctionAttributes {
+    name: string;
+}
+
+interface ComponentFunction {
```

Comment:
superduper nit: you might want to call this `ComponentFunctions` since it's a map of function name to function-attributes!

Reply:
Good call, I was following the pattern of `ComponentField` above, but probably just want to update both those.

Reply:
done!